runtime

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# This file is meant to be sourced by the user, who can then run commands corresponding
+# to the name of the script to be run, e.g.
+#     runtime build                      (to run build.sh)
+#     runtime run                        (to run run.sh)
+#     runtime test integration/tc_68_2   (to run the automated test tc_68_2)
+
+# To source this file to make this function available, issue the command
+#     source runtime.sh
+
+set -e  # Makes any errors exit the script
+
+RUNTIME_DIR=$(dirname "${BASH_SOURCE[0]}")
+SCRIPTS=$(ls $RUNTIME_DIR/scripts/ | awk -F '.' '{ print $1 }')
+
+cd $RUNTIME_DIR
+
+case $1 in
+    build)
+        pushd shm_wrapper && make && popd
+        pushd net_handler && make && popd
+        pushd executor && make && popd
+        pushd dev_handler && make && popd
+        ;;
+
+    run)
+        scripts/run.sh
+        ;;
+    
+
+    test)
+        shift 1
+        scripts/test.sh $@
+        ;;
+
+    format)
+        ;;
+
+    clean)
+        pushd shm_wrapper && make clean && popd
+        pushd net_handler && make clean && popd
+        pushd executor && make clean && popd
+        pushd dev_handler && make clean && popd
+        pushd tests && make clean && popd
```

Comment:
Do you think we should add like `rm /tmp/log-fifo && rm /tmp/challenge.sock && rm /tmp/ttyACM* && rm /dev/shm` to this?

Reply:
the last one is dangerous if other SHM is being used by the OS. I think we can do the first ones though